scanpy/preprocessing/_pca.py

```diff
@@ -52,7 +52,8 @@ def pca(
     zero_center
         If `True`, compute standard PCA from covariance matrix.
         If `False`, omit zero-centering variables
-        (uses :class:`~sklearn.decomposition.TruncatedSVD`),
+        (uses `~sklearn.decomposition.TruncatedSVD` or
+        `dask_ml.decomposition.TruncatedSVD`),
```

Comment:
please don’t remove links for no reason.

```suggestion
        (uses :class:`~sklearn.decomposition.TruncatedSVD` or
        :class:`~dask_ml.decomposition.TruncatedSVD`),
```